app/src/main/java/com/teamagam/gimelgimel/app/common/logging/LoggerFactory.java

```diff
@@ -17,10 +20,12 @@
             new NativeLogger(LoggerFactory.class.getSimpleName());
 
     private static FileWriter sLogWriter;
+    private static String sExternalStorageDirectoryPath;
 
     public static void init(Context context) {
         try {
             sLogWriter = DiskLogger.createLogfileWriter(context);
```

Comment:
get rid of the old disk logger, yours is better. !fix 
feel free to delete the class as well